### PR TITLE
Rebrand/cit filters

### PR DESCRIPTION
**Pull request by ivonne-hernandez**
What (if any) features are you implementing?
- I created a filter component that uses our own filter function.
- I added a `cursor` property to the edit button.

What (if anything) did you refactor?
- I refactored the `CitizenLabels` type and added a `CitizenLabelOptions` type to resolve TS errors.

Were there any issues that arose?
- The `overflow: visible !important` css rule in the `Share` component was causing the `Popover` element to not be anchored to the `Citizenship` button and allow scrolling, so I removed it.

Screenshot of issue in Desktop:
Screenshot 2024-01-18 at 1 40 55 PM
Screenshot of issue in Mobile:
Screenshot 2024-01-18 at 1 39 49 PM

Screenshot of problem in `211`:
Screenshot 2024-01-18 at 1 38 05 PM
Screenshot 2024-01-18 at 1 39 35 PM

Is there anything that you need from your teammate?
- Please review and let me know if you have any questions or concerns.

Desktop views:
Screenshot 2024-01-18 at 1 43 13 PM
Screenshot 2024-01-18 at 1 43 37 PM

### CODE REVIEW

**Review by CalebPena — CHANGES_REQUESTED** [review]

**Review by CalebPena — APPROVED** [review]

**Comment by sydney-devine** [comment]
@ivonne-hernandez Screenshots of the mobile views look off from what I'm seeing? Looks like there's a gap? 

Desktop screens look good. Confirming that the sub bullet checkboxes won't appear until the main one has been checked (green card for example) - so behaving like the filter currently does.

**Comment by CalebPena** [comment]
> @ivonne-hernandez Screenshots of the mobile views look off from what I'm seeing? Looks like there's a gap?
> 
> Desktop screens look good. Confirming that the sub bullet checkboxes won't appear until the main one has been checked (green card for example) - so behaving like the filter currently does.

I think that was an example of a bug that she fixed.

**Comment by sydney-devine** [comment]
> > @ivonne-hernandez Screenshots of the mobile views look off from what I'm seeing? Looks like there's a gap?
> > Desktop screens look good. Confirming that the sub bullet checkboxes won't appear until the main one has been checked (green card for example) - so behaving like the filter currently does.
> 
> I think that was an example of a bug that she fixed.

Can I get final screenshots pls?

**Comment by ivonne-hernandez** [comment]
> > > @ivonne-hernandez Screenshots of the mobile views look off from what I'm seeing? Looks like there's a gap?
> > > Desktop screens look good. Confirming that the sub bullet checkboxes won't appear until the main one has been checked (green card for example) - so behaving like the filter currently does.
> > 
> > 
> > I think that was an example of a bug that she fixed.
> 
> Can I get final screenshots pls?

@sydney-devine here are the screenshots for mobile:
Screenshot 2024-01-25 at 11 47 38 AM
Screenshot 2024-01-25 at 11 47 23 AM

**Comment by ivonne-hernandez** [comment]
@ivonne-hernandez Fix the height when all filters are selected

File: src/Components/Results/Programs/Filter.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  padding: 0.75rem;
+}
+
+.flex-direction-row {
+  display: flex;
+  flex-direction: row;
+}
+
+.MuiFormControlLabel-root.vertical-align {
+  vertical-align: top;
```
Comment (CalebPena) [inline src/Components/Results/Programs/Filter.css:68]: I don't think you need this.
```suggestion
```

File: src/Components/Confirmation/Confirmation.css [hunk 2]
```diff
@@ -39,6 +39,10 @@ svg.confirmation-icon circle {
   transform: translate(50%, -50%);
 }
 
+.edit-button:hover {
```
Comment (CalebPena) [inline src/Components/Confirmation/Confirmation.css:42]: Do we want to merge this change before the redesign is done?

**Review by CalebPena — APPROVED** [review]

**Review by mexi-cano — APPROVED** [review]

**Comment by CalebPena** [comment]
@ivonne-hernandez can you pull from the `feat/results_redesign` branch

**Pull request merged**